### ISSUE TITLE
the kubectl disappeared unexpectly

### ISSUE THREAD

**Issue by jiangxiaoqiang** [lensapp/lens#3570]
**Describe the bug**

when I using lens to edit deployments, the kubectl disappeared automaticlly.

**To Reproduce**
1. Go to  /Users/dolphin/Library/Application Support/Lens/binaries/kubectl/1.21.1
2. download the kubectl:  wget -c https://storage.googleapis.com/kubernetes-release/release/v1.21.1/bin/linux/amd64/kubectl
3. add execute privillige: chmod +x kubectl
4. edit the deployment in the UI
5. the kubectl disappeared in the  /Users/dolphin/Library/Application Support/Lens/binaries/kubectl/1.21.1 folder

**Expected behavior**

**Screenshots**

**Environment (please complete the following information):**
- Lens Version: 5.1.3
- OS: macOS big sur
- Installation method (e.g. snap or AppImage in Linux):

**Comment by jiangxiaoqiang** [comment]
I close the autodownload switch and fix this problem, but could you check if there already have a kubectl in that folder? or give your tips replace or not?

**Comment by msa0311** [comment]
Yes, this works as designed. Lens will remove kubectl binary if autodownload is active.

We just try to understand why Lens is not able to download kubectl binary but it's possible using wget. Are you having some firewall issues or a proxy in between that is only used when working with wget?

Thanks for your help!

**Comment by jiangxiaoqiang** [comment]
I think kubectl binary could not download maybe treated as a normal situation, so we should handle this as a exception workflow.If download failed, just give some tips to tell user(tell the user download failed or tell user to download it using other way). the kubectl not avaliable in every region. If in china, the kubectl official download link was blocked by default. so download failed is a normal situation.

**Comment by Nokel81** [comment]
If you download the `kubectl` someplace else and specify that path in preferences under "Path to kubectl binary" does that work?
